src/aim/pytorch/models.py

```diff
@@ -82,7 +84,91 @@ def forward(
         return tokens, features
 
 
-class AIM(mixins.AIMMixin, nn.Module):
+class AttrDict(dict):
+    def __init__(self, *args, **kwargs):
+        super(AttrDict, self).__init__(*args, **kwargs)
+        self.__dict__ = self    
+
+
+class AIMForImageClassification(nn.Module, PyTorchModelHubMixin):
```

Comment:
Unless I'm missing something, this should work without re-defining the `forward`, etc.:
```python
class AIMForImageClassification(mixins.AIMMixin, PyTorchModelHubMixin, nn.Module):
    ...
```

Reply:
Also please don't forget to add the `AIMForImageClassification` to `__all__`